src/App.jsx

```diff
@@ -14,33 +16,106 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const NOT_SORTED = 'not sorted';
+const SORT_ALPHABETICALLY = 'sort alphabetically';
+const SORT_BY_LENGTH = 'sort by length';
```

Comment:
if you know how to use `enum` you can do apply it. Or just use a object

```suggestion
const sortStatus = {
  alphabetically: 'alphabetically',
  length: 'length',
  default: ''
}
```